study-builder/studies/shared/family-history/family-history-top-level.conf

```diff
@@ -0,0 +1,313 @@
+  GRANDPARENTS_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "Grandparents"
+  },
+  ABOUT_YOU_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "About You"
+  },
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": "Family History of Cancer"
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": "Family History of Cancer"
+    }
+  ],
+  "translatedSubtitles": [
+    {
+      "language": "en",
+      "text": ""
+    }
+  ],
+  "translatedSummaries": [
+    {
+      "statusCode": "CREATED",
+      "language": "en",
+      // Needs to be updated
+      "text": """Completing the Family History of Cancer will help us understanding what role your genetics have had in your disease.."""
+    },
+    {
+      "statusCode": "IN_PROGRESS",
+      "language": "en",
+      // Needs to be updated
+      "text": """Family history in progress!!!."""
+    },
+    {
+      "statusCode": "COMPLETE",
+      "language": "en",
+      "text": "Thank you for providing information regarding your family history of cancer. Please review and update if necessary."
+    }
+  ],
+  // Needs to be updated . Probably will not be needed
+  "readonlyHintTemplate": {
+    "templateType": "HTML",
+    "templateText": "$fh_readonly_hint",
+    "variables": [
+      {
+        "name": "fh_readonly_hint",
+        "translations": [
+          {
+            "language": "en",
+            "text": """<p class="no-margin">
+              Thank you for completing the Family History. If you would like to make any changes, please reach out to the study team
+              at <a href="mailto:info@osproject.org" class="Link">info@osproject.org</a>, or call us
+              at <a href="tel:651-602-2020" class="Link">651-602-2020</a>.
+              </p>"""
+          }
+        ]
+      }
+    ]
+  },
+
+  "introduction": null,
+  "sections": [
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "Introduction",
+        "variables": []
+      },
+      "icons": [],
+      "blocks": [
+        {include required("snippets/family-history-intro-graphic.conf")},
+        {include required("snippets/family-history-section-intro.conf")}
+      ]
+    },
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "Instructions",
+        "variables": []
+      },
+      "icons": [],
+      "blocks": [
+        {include required("snippets/family-history-section-instructions.conf")}
+      ]
+    },
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "Parents",
+        "variables": []
+      },
+      "icons": [],
+      "blocks": [
+        ${PARENT_SECTION_INTRO_BLOCK},
+        {
+          "activityCode": "FAMILY_HISTORY_MOTHER",
+          "createActivityInstance": true,
```

Comment:
Just a note that we don't have this property!

Reply:
Removing 34c4f975